src/onnx/parse_matmul.cpp

```diff
@@ -66,9 +66,8 @@ struct parse_matmul : op_parser<parse_matmul>
                     make_op("multibroadcast", {{"out_lens", l1_broadcasted_lens}}), l1);
             }
         }
-
-        auto dot_res =
-            info.add_instruction(make_op(opd.op_name, {{"alpha", 1}, {"beta", 0}}), bl0, bl1);
+        instruction_ref dot_res;
+        dot_res          = info.add_instruction(make_op(opd.op_name), bl0, bl1);
```

Comment:
Why is this two lines? It should just be `auto dot_res = info.add_instruction(make_op(opd.op_name), bl0, bl1)`.

Reply:
I think it was automatic formatting. Let me check though.